packages/gatsby/src/utils/api-runner-node.js

```diff
@@ -245,15 +244,6 @@ module.exports = async (api, args = {}, pluginSource) =>
       apiSpan.setTag(key, value)
     })
 
-    // Check that the API is documented.
-    // "FAKE_API_CALL" is used when code needs to trigger something
-    // to happen once the the API queue is empty. Ideally of course
-    // we'd have an API (returning a promise) for that. But this
-    // works nicely in the meantime.
-    if (!apiList[api] && api !== `FAKE_API_CALL`) {
```

Comment:
This was kind of a strange one -- I think I can safely remove this?

Reply:
Maybe? See https://github.com/gatsbyjs/gatsby/pull/11831 - `FAKE_API_CALL` was added as workaround for issues when page is deleted. I think you could try `gatsby develop` in `gatsby-starter-blog` and try to delete one of markdown posts and see what happens :P